src/components/portrait/portrait-gravatar.component.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MD5 from 'crypto-js/md5';
+
+class PortraitGravatar extends React.Component {
+  static propTypes = {
+    /**
+     * A custom class name for the component.
+     */
+    className: PropTypes.string,
+
+    /**
+     * The user's email address for the Gravatar.
+     */
+    gravatarEmail: PropTypes.string.isRequired,
+
+    /**
+     * The dimensions (size) to render the Gravatar, in pixels.
+     */
+    dimensions: PropTypes.number.isRequired,
+
+    /**
+     * Defines the alt HTML string.
+     */
+    alt: PropTypes.string
+  }
+
+  /**
+   * Gets src url based on passed gravatar email
+   *
+   * @method gravatarSrc
```

Comment:
We don't need to keep `@method gravatarSrc` and `@return {String}`. I think we can remove it and just keep the comments
```
  /** Gets src url based on passed gravatar email */
```

Reply:
@dawidzarzycki: I've now cleaned up the JSDoc blocks in e24f1e8 :thumbsup: